fcc-test-suite-template-master(1)/fcc-test-suite-template-master/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <div id="top-page">
+        <h1 id="title">What's your favorite cat?</h1>
+        <p id="description">Thank you for your time!!</p>
+      </div>
+      <div id ='forms'>
+
+        <form id='survey-form'>
+          <div id="layout-name" class="forms1">
+            <label id="name-label" for ='name'>Enter Your Name:</label><br>
+            <input type='text' class="borders" id='name' placeholder="Name" required>
+          </div>
+          <div id="layout-email" class="forms1">
+            <label id="email-label" for ='email'>Enter Your Email:</label><br>
+            <input type='email' class="borders" id='email' placeholder="Email" required>
+          </div>
+          <div id="layout-number" class="forms1">
+            <label id="number-label" for ='number'>Enter Your Age:</label><br>
+            <input type='number' class="borders" min="18" max="100" id='number' placeholder="Age" required>
+          </div>
+          <div id="layout-dropdown">
+            <h3 id="title2">What's your favorite type of cat?</h3>
```

Comment:
Same here, instead of `tittle2` kindly use a class name that is more descriptive